src/operator/nn/mkldnn/mkldnn_pooling-inl.h

```diff
@@ -92,12 +92,17 @@ inline bool SupportMKLDNNPooling(const PoolingParam &param,
 
   if (param.pooling_convention == pool_enum::kValid)
     return true;
+  else
+    return false;
 
+// need to support pooling convention full 
```

Comment:
Please add a link to the issue as part of the comment

Reply:
you mean , link to this PR ? or JIRA ?

Reply:
To jira